le_utils/constants/format_presets.py

```diff
@@ -54,6 +54,11 @@
 HTML5_DEPENDENCY_ZIP = "html5_dependency"
 HTML5_DEPENDENCY_ZIP_READABLE = "HTML5 Dependency (Zip format)"
 
+H5P_ZIP = "h5p_zip"
```

Comment:
This should be `h5p` to match the key in `resources/presetlookup.json` 

caught by uncle Travis

<img width="810" alt="Screen Shot 2019-08-07 at 6 41 35 PM" src="https://user-images.githubusercontent.com/163966/62662746-089d8880-b943-11e9-8194-726744d9222f.png">